.env

```diff
@@ -1,6 +1,6 @@
 REACT_APP_UI_ILS_VERSION=$npm_package_version
 REACT_APP_INVENIO_VERSION=1.0.0a13
-REACT_APP_INVENIO_UI_URL=https://localhost:5000
-REACT_APP_INVENIO_REST_ENDPOINTS_BASE_URL=https://localhost:5000/api
+REACT_APP_INVENIO_UI_URL=https://127.0.0.1:5000
+REACT_APP_INVENIO_REST_ENDPOINTS_BASE_URL=https://127.0.0.1:5000/api
```

Comment:
Out of curiosity, why this change?

Reply:
it was creating issues with the CSRF token for me, backend running on 127.0.0.1 and react app on localhost (or the opposite)